parachain/pallets/basic-channel/src/outbound/test.rs

```diff
@@ -97,38 +91,40 @@ pub fn new_tester() -> sp_io::TestExternalities {
 
 fn run_to_block(n: u64) {
 	while System::block_number() < n {
-			System::set_block_number(System::block_number() + 1);
-			BasicOutboundChannel::on_initialize(System::block_number());
+		System::set_block_number(System::block_number() + 1);
+		// TODO: is this necessary for the test? It's mentioned in the unit testing docs:
+		// https://docs.substrate.io/main-docs/test/unit-testing/
+		System::on_initialize(System::block_number());
```

Comment:
The tests run fine without this call, and it looks like this implementation in `System` doesn't do anything. Is it worth keeping it anyway?

Reply:
yeah lets keep it.